src/ApparentHorizons/StrahlkorperInDifferentFrame.hpp

```diff
@@ -26,7 +26,7 @@ class not_null;
 /// The destination Strahlkorper has the same l_max() and m_max() as the
 /// source Strahlkorper.
 ///
-/// Note that because the Blocks inside the Domain allow access to
+/// \note Because the Blocks inside the Domain allow access to
 /// maps only between a selected subset of frames, we cannot use
 /// strahlkorper_in_different_frame to map between arbitrary frames;
 /// allowing strahlkorper_in_different_frame to work on more frames
```

Comment:
Can you add to this note that this function *only* works for SrcFrame = Grid and DestFrame = Inertial.